vendor/github.com/stretchr/testify/assert/assertions.go

```diff
@@ -876,13 +1155,14 @@ func InEpsilonSlice(t TestingT, expected, actual interface{}, epsilon float64, m
 //
 //   actualObj, err := SomeFunction()
 //   if assert.NoError(t, err) {
-//	   assert.Equal(t, actualObj, expectedObj)
+//	   assert.Equal(t, expectedObj, actualObj)
 //   }
-//
-// Returns whether the assertion was successful (true) or not (false).
 func NoError(t TestingT, err error, msgAndArgs ...interface{}) bool {
+	if h, ok := t.(tHelper); ok {
+		h.Helper()
```

Comment:
Its panicking here for some reason 🤔

Reply:
oh! I see it we're passing `nil` as argument here 🤔 ; https://github.com/docker/swarmkit/blob/cb78da37418bbee263ecb6e8ab4c83b51a93b8dc/manager/role_manager_test.go#L109

Reply:
https://github.com/docker/swarmkit/blob/486ad69951868792367a503042ef3f55b399cc40/ca/testutils/cautils.go#L102-L106